netlify.toml

```diff
@@ -0,0 +1,3 @@
+[build]
+  command = "next build"
+  publish = ".next"
```

Comment:
Please add a newline at the end of file.

Reply:
It's still stating no newline at end of file. [See this](https://github.com/TypeSafe-Travellers/App/pull/11/commits/118daa20ae589c47991ccc3c3d690fd1e831b9e8)

Reply:
--ignore--unknown in prettier scripts was ignoring the .toml file. Should be fixed now